src/sentry/workflow_engine/migrations/0060_rename_azure_devops_action_to_vsts.py

```diff
@@ -0,0 +1,71 @@
+import logging
+
+from django.apps.registry import Apps
+from django.conf import settings
+from django.db import migrations, router, transaction
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.utils import redis
+from sentry.utils.iterators import chunked
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBarApprox
+
+logger = logging.getLogger(__name__)
+
+CHUNK_SIZE = 1000
+
+
+def rename_azure_devops_action_to_vsts(apps: Apps, schema_editor: BaseDatabaseSchemaEditor) -> None:
+    Action = apps.get_model("workflow_engine", "Action")
+
+    backfill_key = "workflow_engine_rename_azure_devops_action_to_vsts"
+    redis_client = redis.redis_clusters.get(settings.SENTRY_MONITORS_REDIS_CLUSTER)
+    progress_id = int(redis_client.get(backfill_key) or 0)
+
+    for actions in chunked(
+        RangeQuerySetWrapperWithProgressBarApprox(
+            Action.objects.filter(id__gt=progress_id, type="azure_devops"), step=CHUNK_SIZE
+        ),
+        CHUNK_SIZE,
```

Comment:
It seems like we've been doing this lately so it's ok, but question for @wedamija (or Raj if you've already asked this) whenever you get to it - why is it that we used to _not_ filter in migrations and always would iterate over all the rows of a model, but now it's ok? Is it because we're chunking it? This used to be a red flag to me so I wanna know what's changed.

Reply:
Just saw this comment, you're right that it's better to not filter in the query: https://github.com/getsentry/sentry/pull/91496#discussion_r2087345763

To some extent it depends on table size. If you're dealing with a 100k row table it's probably not going to matter. But for a 100m row table it definitely will. Generally it's just safer to do what you said, and filter in code.